game/gothic.cpp

```diff
@@ -660,6 +678,8 @@ void Gothic::settingsSetS(std::string_view sec, std::string_view name, std::stri
 float Gothic::settingsGetF(std::string_view sec, std::string_view name) {
   if(name.empty())
     return 0;
+  if(instance->modFile != nullptr && instance->modFile->has(sec,name))
```

Comment:
modFile not suppose to have any settings, therefore is not part of init-chain. See how `wrldDef` is implemented.

Reply:
I wouldn't have a way to get the OutputUnits value then, should I have an `ouDef` too?

Reply:
same as with world?
`std::string_view defaultOutputUnits() const`

Reply:
Ok, that's a yes, then. WIll do.

Reply:
Done